app/routes/member/invites/$invitationId/index.tsx

```diff
@@ -34,6 +34,13 @@ export default function MemberInvitationIdRoute() {
   const [guestsInput, setGuestsInput] = useState(
     invitation.guests ? invitation.guests : 0
   );
+  const [yesIsChecked, setYesIsChecked] = useState(() => {
+    if (invitation.status === "RESPONDED_YES") {
```

Comment:
Whenever you have something that fits this pattern:
```
if(something) { return true } else { return false}
```
you can replace it with
```
return something
```
A bit shorter and just as clear is a win-win to me :)

Reply:
Well, that is assuming that `something` is already a boolean. If it's not, you can just do `Boolean(something)`